### PR TITLE
Add Clock Manager Introduction

### PR DESCRIPTION
**Pull request by djzage**
Created a Clock Manager overview README to give a better understanding of the library. Linked the overview to associated information in the directory.  Also created a callout on the primary project readme.

### CODE REVIEW

**Comment by yoongsiang2** [comment]
@djzage thanks for your PR. Can you help to add your signoff in the commit msg and add "clkmgr:" tag in front of the commit title?

File: README.md [hunk 1]
```diff
@@ -138,6 +138,13 @@ but **NOT** with the Go wrapper, which may break!
   * Options in opt.h - Parse pmc tool command line parameters
   * Init in init.h - Initialize objects for a pmc tool
 
+# <u>Clock Manager</u>
+Leveraging the libptpmgmt Library, the clock Manager is a framework for monitoring network time
+synchronization on the local platform. The Clock Manager exposes an API to
+enable reporting time synchronization status to any application executing on the
+system. It is planned that Clock Manager will not only be able to report
+timing telemetry information, but also configure the platform.
```
Comment (yoongsiang2) [inline README.md:146]: maybe we can add a link to the clock manager main readme file, using something like below:
\[Clock Manager](./clkmgr/README.md)

**Review by yoongsiang2 — CHANGES_REQUESTED** [review]

File: clkmgr/README.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+Applications utilizing Clock Manager must be allowed to be licensed and released
+under any free or non-free terms. The time synchronization daemon (e.g., Linux
+PTP or Chrony) are licensed under GPL. The Clock Manager framework insulates the
+application from any requirement to release the source code. The Clock Manager
+service is implemented using a process separate from both the application and
+time synchronization daemon. This process is the Clock Manager Proxy (CMP). The
+CMP code will include GPL and/or link to GPL code making it a derivative and
+requiring that all code in the CMP code be licensed under the GPLv2. Any Clock
+Manager support libraries must be released under a GPL-compatible license that
+can also be used with proprietary software, such as MIT or modified BSD
```
Comment (yoongsiang2) [inline clkmgr/README.md:72]: We decided to use BSD-3-Clause license, so we can explicitly mention the license.

**Review by yoongsiang2 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (djzage) [reply README.md]: Updated!

File: clkmgr/README.md (hunk 2 above)
Comment (djzage) [reply clkmgr/README.md:73]: Updated

**Review by yoongsiang2 — APPROVED** [review]
LGTM. Will merge this once all CI checks passed.

**Review by JunAnnLaiIntel — APPROVED** [review]
LGTM

**Pull request merged**